src/components/GenerateGifForm.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import './GenerateGifForm.css';
+
+class GenerateGifForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { name: '', titleText: '' };
```

Comment:
@colebillys19 @lopezh18 @phillipson808 - since we're storing this info in redux, this component shouldn't need any local component state. (@colebillys19 you don't need to worry about this for now, Haley and Spencer are working on it)

Reply:
Done